modules/yardstick/src/main/java/org/apache/ignite/yardstick/cache/load/IgniteCacheRandomOperationBenchmark.java

```diff
@@ -465,15 +545,64 @@ private void executeRandomOperation(IgniteCache cache) throws Exception {
 
             case SQL_QUERY:
                 doSqlQuery(cache);
+                break;
+
+            case CONTINUOUS_QUERY:
+                doContinuousQuery(cache, map);
         }
+        storeStatistics(cache.getName(), map, op);
+    }
+
+    /**
+     * @param cacheName Ignite cache name.
+     * @return Operation.
+     */
+    @NotNull private Operation nextRandomOperation() {
+        return allowOperations.get(nextRandom(allowOperations.size()));
+    }
+
+    /**
+     * @param cacheName Ignite cache name.
+     * @param map Parameters map.
+     * @param op Operation.
+     */
+    private void storeStatistics(String cacheName, Map<Object, Object> map, Operation op) {
+        String opCacheKey = String.format("%s_%s", op, cacheName);
```

Comment:
It's better to form this key the following way cacheName + "_" + opName. It will be faster then relying on the formatter.